messages/store.json

```diff
@@ -127,21 +127,24 @@
             "extensionPointNameDescription": "Means of identifying entry points within the domain code that are capable of being extended by partners.Takes the form domain_sub-domain_interface. E.g: checkout_summary_computeTax",
             "regExtensionNameFlagDescription": "Unique name for extension",
             "errEPN": "Invalid EPN. Run [commerce:extension:points:list] for all valid EPN values",
-            "savingConfigIntoConfig": "Sucessfully registered the extension. Use command [commerce:extension:map] to link the extension to a store.",
+            "savingConfigIntoConfig": "Successfully registered the extension. Use command [commerce:extension:map] to link the extension to a store.",
             "apexNameSpaceFlagDescription": "Optional namespace for apex class",
             "undefinedName": "Undefined registered extension name. Enter '-r <name>'"
         },
         "map": {
             "cmdDescription": "Maps an extension to a specific webstore",
-            "StoreNameFlagDescription": "The name of the web store to map the extension",
-            "errStoreName": "Web-storename is not valid",
-            "errStoreId": "Store Id is not valid",
-            "nameAlreadyExists": "Registered extension name already exists: ",
+            "StoreNameFlagDescription": "The name of the webstore to map the extension",
+            "errStoreName": "Can't find webstore with the name:",
+            "errStoreId": "Can't find store ID:",
+            "error": "Error while mapping the given registered extension name:",
             "extensionPointNameDescription": "Means of identifying entry points within the domain code that are capable of being extended by partners.Takes the form domain_sub-domain_interface. E.g: checkout_summary_computeTax",
             "regExtensionNameFlagDescription": "Unique name for extension",
-            "savingConfigIntoConfig": "Sucessfully mapped apex class to store.",
-            "storeId": "Either name or id should be provided"
+            "savingConfigIntoConfig": "Successfully mapped the extension to webstore:",
+            "storeId": "Optional store ID flag",
+            "nonexistent": "Registered extension name doesn't exist in sObject table RegisteredExternalService.",
```

Comment:
what do you think if we follow a similar error message as errStoreId? So something like 
`Can't find extension with registered extension name:`?